### DISCUSSION TITLE
We need to be able to report spam.

### DISCUSSION THREAD

**Discussion by 100thCoin** [ludumdare/ludumdare#2142] [Ideas]
Bot comments on games and the feed have been increasing, and it's almost always accompanied with a link to a shady website. Though this post will be featuring examples from my own games, a site-wide solution for reporting and removing such comments is becoming a necessity.

Here are comments left on [my Ludum Dare 46 game](https://ldjam.com/events/ludum-dare/46/morose), and[ Ludum Dare 50 game](https://ldjam.com/events/ludum-dare/50/block-or-bust).

![image](https://user-images.githubusercontent.com/23084831/187827641-0b6e5f16-b6da-4905-a257-fc22d57b55cd.png)

![image](https://user-images.githubusercontent.com/23084831/187827669-69ee0eae-b1d2-4206-a26b-4bf5c0071bcd.png)

The comment on the LD46 game appear to be using words relating to the game, such as "That little slowdown from the packs was just enough to make it out of the crazyiest spaces" except not only does my game not feature any such "slowdowns" or "packs", but there are certainly no "crazy spaces" to be found either. They're simply throwing around buzz words that sound like feedback as a way to maliciously hide their link in what appears to be actual feedback.

While a report button is the first thing that comes to mind, an option to lock a game's comment thread after play & rate ends would also be useful. I honestly don't want any additional comments on my previous games. The feedback from those entries has been taken in long ago, and any extra comments feel redundant. This sort of "thread lock" would be especially useful for games that have full scale releases. Games like [Friday Night Funkin'](https://ldjam.com/events/ludum-dare/47/friday-night-funkin) come to mind, as the popularity of this game has lead to multiple spam messages; both from bots and fans of the game. The bot spam is especially concerning, with fake links that appear to take you to Friday Night Funkin's official page.

![image](https://user-images.githubusercontent.com/23084831/187832487-15808815-4328-4b20-8114-a9c4df02c5e2.png)

There have been a few posts in the ldjam feed about this if you're looking for more info on what everyone else has said already. (And even posts about spam are gathering spam comments)

[Example 1](https://ldjam.com/events/ludum-dare/50/protect-the-light/spam)

[Example 2](https://ldjam.com/events/ludum-dare/50/re-placement/so-about-that-report-feature)

[Example 3](https://ldjam.com/events/ludum-dare/50/garden-guardian/participate-in-ludum-dare-51)

As far as I'm aware, there hasn't been any official word on the matter. If a report button is planned (or my suggested "Lock thread" option) we would like to hear about it.

**Comment by 100thCoin** [comment]
As a brief update, the examples I provided are from a few weeks ago. This example was posted today on [a game submitted for Ludum Dare 41](https://ldjam.com/events/ludum-dare/41/the-fast-dating), almost four and a half years ago.

![image](https://user-images.githubusercontent.com/23084831/187833948-b01f8333-a9e6-470c-916e-0fac58bfa72c.png)

The account even has a profile picture and a bio (with a shady link).

Edit: I'll refrain from posting more examples, as that's not what this discussion is about. I'm creating this discussion to hopefully hear about any measures being taken to implement a report feature.

Second Edit: I know I said I wouldn't post more examples, but this is a serious issue and it needs to be addressed. From [Ludum Dare 50's jam winner](https://ldjam.com/events/ludum-dare/50/shotgun-king) (A game which has also gained popularity in the form of a full release):

![image](https://user-images.githubusercontent.com/23084831/187996294-9b92b420-d466-4638-8342-bf220b3ac66a.png)

I wouldn't be opposed to allowing users to delete comments on their own game's page. This allows them to remove any spam without requiring a report. (perhaps require a user to click a radio button explaining why they removed it? spam / no feedback / other). The only flipside is they could delete genuine feedback that they disagree with (which would be the exact opposite of what you should do with negative feedback). Something needs to be done about this though.

**Comment by mikekasprzak** [comment]
Reporting is coming soon. It's currently pending a frontend code upgrade I've been finishing up, and some additional data gathering. Akamai is helping us track suspicious activity, but that alone wont be enough.

**Reply by TrickyFatCat** [reply]
Is there any news? Currently the site is a landfill of spammers and it's impossible to promote the jam among beginner game developers just because of all spam it has now.

**Comment by 100thCoin** [comment]
It appears the bots are submitting games now.

![_________Botspam](https://user-images.githubusercontent.com/23084831/196061417-189b514e-2ebe-43a7-9022-54e65a85dca0.PNG)
